File: C#_Assignments/DateTimeCalculator/Models/InputEntity.cs
using System;

namespace DateTimeCalculator.Models
{
    public class InputEntity
    {
        public DateTime Date { get; set; }
        public int[] Params { get; set; }

        public DateOperation Operation { get; set; }

        public InputEntity(DateTime date, int[] param, DateOperation operation)
        {
            this.Date = date;
            this.Params = param;
            this.Operation = operation;
        }

    }
    
}

File: week2&3/DateTimeCalculator/src/main/java/com/sapient/date_time_calculator/models/InputEntity.java
package com.sapient.date_time_calculator.models;

import java.time.LocalDate;

public class InputEntity {

	private LocalDate date;
	private int[] params;
	private DateOperation operation;

	public InputEntity(LocalDate date, int[] params, DateOperation operation) {
		super();
		this.date = date;
		this.params = params;
		this.operation = operation;
	}

	public LocalDate getDate() {
		return date;
	}

	public void setDate(LocalDate date) {
		this.date = date;
	}

	public int[] getParams() {
		return params;
	}

	public void setParams(int[] params) {
		this.params = params;
	}

	public DateOperation getOperation() {
		return operation;
	}

	public void setOperation(DateOperation operation) {
		this.operation = operation;
	}

}
